packages/dev/docs/pages/releases/2023-10-06.mdx

```diff
@@ -0,0 +1,102 @@
+- Add docs for links in collections and client side routing guide - [@devongovett](https://github.com/devongovett) - [PR](https://github.com/adobe/react-spectrum/pull/5078)
+- Move React Aria dnd introduction to concepts section - [@devongovett](https://github.com/devongovett) - [PR](https://github.com/adobe/react-spectrum/pull/5088)
+
+## Under construction
+
+Fixes to pre-released components are listed below. Please feel free to try them out, and report any issues you encounter.
+
+**React Aria Components** (Beta)
+- Allow FileTrigger to accept the same file twice - [@yihuiliao](https://github.com/yihuiliao) - [PR](https://github.com/adobe/react-spectrum/pull/5045)
+- Update RAC Link API - [@devongovett](https://github.com/devongovett) - [PR](https://github.com/adobe/react-spectrum/pull/5074)
+- Use PressResponder in RAC overlays and document MenuTrigger long press - [@devongovett](https://github.com/devongovett) - [PR](https://github.com/adobe/react-spectrum/pull/5065)
+- Filter data attributes - [@iamwillpowell](https://github.com/iamwillpowell) - [PR](https://github.com/adobe/react-spectrum/pull/5008)
+- Fix typeahead in RAC ListBox - [@devongovett](https://github.com/devongovett) - [PR](https://github.com/adobe/react-spectrum/pull/5146)
+- Advanced customization guide - [@devongovett](https://github.com/devongovett) - [PR](https://github.com/adobe/react-spectrum/pull/5066)
+- Minor doc improvements - [@devongovett](https://github.com/devongovett) - [PR](https://github.com/adobe/react-spectrum/pull/5101)
+- Fix specificity of Tailwind generated selectors - [@devongovett](https://github.com/devongovett) - [PR](https://github.com/adobe/react-spectrum/pull/5147)
+- Refactor internal contexts to expose state publicly - [@devongovett](https://github.com/devongovett) - [PR](https://github.com/adobe/react-spectrum/pull/5052)
+- Update Breadcrumb Link section in docs - [@LFDanLu](https://github.com/LFDanLu) - [PR](https://github.com/adobe/react-spectrum/pull/5083)
+- Add missing DatePicker states - [@devongovett](https://github.com/devongovett) - [PR](https://github.com/adobe/react-spectrum/pull/5158)
+- Add docs for the Group component - [@devongovett](https://github.com/devongovett) - [PR](https://github.com/adobe/react-spectrum/pull/5060)
+
+**React Spectrum DropZone** (Alpha)
+- React Spectrum DropZone docs - [@yihuiliao](https://github.com/yihuiliao) - [PR](https://github.com/adobe/react-spectrum/pull/5076)
+- Add chromatic stories to React Spectrum DropZone - [@yihuiliao](https://github.com/yihuiliao) - [PR](https://github.com/adobe/react-spectrum/pull/5053)
+
+**React Spectrum Pending Button** (Beta)
+- Remove UNSAFE_ from isPending prop - [@iamwillpowell](https://github.com/iamwillpowell) - [PR](https://github.com/adobe/react-spectrum/pull/5150)
+- Add isPending to Button docs - [@iamwillpowell](https://github.com/iamwillpowell) - [PR](https://github.com/adobe/react-spectrum/pull/5150)
+
+What do we want to do with these commits?
```

Comment:
These are the commits that I didn't know where to put them. They can basically be broken down into three categories:
1. Upgrades to node/user event/CSS
2. Fixing the verdaccio build
3. Translations

Reply:
Some discretion can be applied, we don't have to name every commit/PR, especially when they are mostly tooling updates or internal fixes to example apps